website/src/word-panel.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react"
+import { Disclosure, DisclosureContent, useDisclosureState } from "reakit/Disclosure"
+import * as Dailp from "src/graphql/dailp"
+import { BasicMorphemeSegment, TagSet, ViewMode } from "./types"
+import * as css from "./word-panel.css"
+import { FormAudio } from "./audio-player"
+import { MorphemicSegmentation } from "./segment"
+import { Button } from "reakit/Button"
+
+export interface wordPanelDetails {
+    currContents: Dailp.FormFieldsFragment | null
+    setCurrContents: (currContents: Dailp.FormFieldsFragment | null) => void
+}
+
+export const WordPanel = (
+    p : {
+        segment : Dailp.FormFieldsFragment | null
+        setContent : (content: Dailp.FormFieldsFragment | null) => void
+        viewMode : ViewMode
+        onOpenDetails: (morpheme: BasicMorphemeSegment) => void
+        tagSet: TagSet
+    }
+) => {
+    if (p.segment) {
+        const disclosure = useDisclosureState({ visible: true });
+        const translation = p.segment.englishGloss.join(", ")
+        return (
+            <div className={css.wholePanel}>
+                
+                <div 
+                className={css.wordPanelContent}>
+                    <Button className={css.wordPanelButton} onClick={() => p.setContent(null)}>X</Button>
+
+                    <h2>{p.segment.source}</h2> 
+                    {p.segment.simplePhonetics ? (
```

Comment:
Since there's like three different options here based on two conditions, I would extract this out into a flatter structure. Maybe like

```ts
let wordBreakdown = null
if (x) {
  wordBreakdown = ...
} else if (y) {
  wordBreakdown = ...
}

return (
  <div>
    {wordBreakdown}
  </div>
)
```